api/tests/core/offerers/test_models.py

```diff
@@ -91,6 +91,11 @@ def test_at_most_one_virtual_venue_per_offerer(self):
             repository.save(venue3)
         assert err.value.errors["isVirtual"] == ["Un lieu pour les offres numériques existe déjà pour cette structure"]
 
+    def test_physical_venue_must_have_an_offerer_address(self):
+        with pytest.raises(IntegrityError) as err:
+            factories.VenueFactory(siret="siret", offererAddress=None)
```

Comment:
Détail : je ne suis pas fan de `"siret"` dans le SIRET, un jour il faudra quand même qu'on ajoute une contrainte sur le format. De toute façon, `VenueFactory` est déjà un lieu avec SIRET : 
```suggestion
            factories.VenueFactory(offererAddress=None)
```